custom/icds_reports/migrations/sql_templates/create_tables_and_views.sql

```diff
@@ -55,6 +55,9 @@ CREATE OR REPLACE VIEW awc_location_months AS
 	months.month_name AS month_display
   FROM awc_location awc_location
   CROSS JOIN "icds_months" months;
+CREATE INDEX "awc_location_months_indx1" ON "awc_location_months" (aggregation_level, month);
+CREATE INDEX "awc_location_months_indx2" ON "awc_location_months" (aggregation_level, state_site_code, month);
```

Comment:
if you change the order of these I think you can combine this with the previous one:
```
aggregation_level, month, state_site_code
```